torchgeo/datasets/loveda.py

```diff
@@ -0,0 +1,261 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""LoveDA dataset."""
+
+import glob
+import os
+from typing import Callable, Dict, List, Optional
+
+import numpy as np
+import torch
+from PIL import Image
+from torch import Tensor
+
+from .geo import VisionDataset
+from .utils import download_and_extract_archive
+
+
+class LoveDA(VisionDataset):
+    """LoveDA dataset
+
+    The LoveDA <https://github.com/Junjue-Wang/LoveDA> datataset is a semantic segmentation
+    dataset.
+
+    Dataset features:
+
+    * 2713 urban scene and 3274 rural scene HSR images with a total of 166768 annotated objects
+    from Nanjing, Changzhou and Wuhan cities
+    * dataset comes with predefined train, validation, and test set
+
+    Dataset format:
+
+    * images are three-channel pngs with dimension 1024x1024
+    * segmentation masks are single-channel pngs
+
+    Dataset classes:
+
+    1. background
+    2. building
+    3. road
+    4. water
+    5. barren
+    6. forestn
+    7. agriculture
+
+    No-data regions assigned with 0 and should be ignored.
+
+    If you use this dataset in your research, please cite the following paper:
+
+    * <https://openreview.net/forum?id=bLBIbVaGDu>
+    """
+
+    # url of train split
+    url = "https://drive.google.com/file/d/1xbnKVN5aRMlpxISXgutzQO0hPT_b4lMi/view?usp=sharing"
+    # md5 of train split
+    md5 = "de2b196043ed9b4af1690b3f9a7d558f"
```

Comment:
`url` and `md5` don't seem to be required since they're overriden by values in info_dict